nbviewer/handlers.py

```diff
@@ -100,10 +99,11 @@ def init_handlers(formats, providers, base_url, localfiles):
         (r'/(robots\.txt|favicon\.ico)', web.StaticFileHandler)
     ]
 
-    handlers = provider_handlers(providers)
-
     # Add localfile handlers if the option is set
-    handlers = [(r'/localfile/?(.*)', LocalFileHandler)]+handlers if localfiles else handlers
```

Comment:
This logic prepended the localfile handlers. I believe this was required to get the precedence right for certain cases, maybe slideshow for local notebooks. The change below appears to put the localfile handlers last, which may cause problems.

Reply:
I've updated the code to insert localfile first. That should be functionally equivalent to the original.